SofaKernel/framework/sofa/defaulttype/CMakeLists.txt

```diff
@@ -32,6 +32,12 @@ set(HEADER_FILES
     Color.h
 )
 
+if(EIGEN3_FOUND OR Eigen3_FOUND)
```

Comment:
Why `Eigen3_FOUND`? I only see `EIGEN3_FOUND` in https://github.com/sofa-framework/sofa/blob/master/SofaKernel/cmake/Modules/FindEigen3.cmake#L63

Reply:
Eigen3_FOUND is the syntax which actually respect the most the "cmake" standard for package variable definition. We redistribute a FindEigen.cmake, but there could be other with a more conforming syntax for cmake variables.

For instance when you do a find_package(SofaFramework) the variable `SofaFramework_FOUND` is set to true upon success and false upon failure. The variable SOFAFRAMEWORK_FOUND does not exist.

Reply:
Interesting. Was not aware of that.